pkg/apis/middleware/session.go

```diff
@@ -25,6 +25,7 @@ func CreateTokenToSessionFunc(verify VerifyFunc) TokenToSessionFunc {
 			Verified          *bool    `json:"email_verified"`
 			PreferredUsername string   `json:"preferred_username"`
 			Groups            []string `json:"groups"`
+			Acr               string   `json:"acr"`
```

Comment:
As it is an acronym I would prefer this to be capitalized.
```suggestion
			ACR               string   `json:"acr"`
```